cli/tests/pcluster/validators/test_monitoring_validators.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+
+from pcluster.config.cluster_config import CloudWatchLogs, LogRotation, Logs
+from pcluster.validators.monitoring_validators import LogRotationValidator
+from tests.pcluster.validators.utils import assert_failure_messages
+
+
+@pytest.mark.parametrize(
+    "logs, expected_message",
+    [
+        (Logs(cloud_watch=CloudWatchLogs(enabled=True), rotation=LogRotation(enabled=True)), None),
+        (Logs(cloud_watch=CloudWatchLogs(enabled=False), rotation=LogRotation(enabled=False)), None),
+        (Logs(rotation=LogRotation(enabled=False)), None),
+        (Logs(rotation=LogRotation(enabled=True)), None),
+        (
+            Logs(cloud_watch=CloudWatchLogs(enabled=False), rotation=LogRotation(enabled=True)),
+            "Cloudwatch Logs is disabled but Log Rotation is enabled. Logs will be rotated and removed from the "
```

Comment:
Nitpick: Would it read better to say `CloudWatch Logging is...`

Reply:
Good idea, changed